src/actions/collect.py

```diff
@@ -187,10 +205,10 @@ def evaluate_optimizations(self, connection, cur, original_query):
 
             # check that execution plan is unique
             evaluate_sql(cur, optimization.get_heuristic_explain())
-            execution_plan = database.get_execution_plan(
+            optimization.basic_execution_plan = database.get_execution_plan(
```

Comment:
We should use EXPLAIN (costs off) so tests are more stable. Small cost changes that do not flip the query plan should not make the tests fail.